README.md

```diff
@@ -1 +1,185 @@
-# hello-rails-react
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Kindly remove this logo code. You can use your personal logo. Do this for all your future projects.